cabot/cabotapp/tasks.py

```diff
@@ -63,6 +63,8 @@ def run_status_check(pk):
 def run_all_checks():
     checks = models.StatusCheck.objects.filter(active=True).all()
     for check in checks:
+        if check.use_activity_counter and check.activity_counter <= 0:
+            continue
```

Comment:
Could be useful to add logging here in case we need to debug issues where a check isn't running when we think it should be

Reply:
Good idea.  Is `info` log level ok?